cpp/oneapi/dal/algo/decision_forest/common.hpp

```diff
@@ -118,9 +118,9 @@ enum class voting_mode {
 
 /// Available splitting strategies for building trees
 enum class splitter_mode {
-    /// Treshold for node is chosen as best among all bins
+    /// Treshold for a node is chosen as the best among all bins
     best,
-    /// Treshold for node is chosen as random treshold
+    /// Treshold for a node is chosen as random treshold
```

Comment:
***Threshold for a node is the best for a set chosen at random***

English and clarification.